keyboards/rotr/info.json

```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "rotr", 
+    "url": "https://github.com/kwlodarczy/qmk_firmware/tree/master/keyboards/rotr", 
```

Comment:
[`A URL to the keyboard’s product page, QMK.fm/keyboards page, or other page describing information about the keyboard.`](https://docs.qmk.fm/#/reference_info_json)